src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -34,6 +35,22 @@ protected function setUp()
         $this->builder = new FormBuilder(null, null, $this->dispatcher, $this->factory);
     }
 
+    protected function getExtensions()
```

Comment:
The issue with a separate class, is that you can't add more core extensions, as each one will need a separate class then, which makes extending the classes or using multiple core extensions difficult.

In terms of the overhead, I don't think that is really an issue. The overhead is so small per test that the impact would be negligible